htdocs/core/lib/functions.lib.php

```diff
@@ -40,6 +40,7 @@
  */
 
 include_once DOL_DOCUMENT_ROOT.'/core/lib/json.lib.php';
+include_once DOL_DOCUMENT_ROOT.'/core/lib/signature.lib.php';
```

Comment:
The functions.lib.php is a file used everywhere.
signature should not be included here or it will be loaded everywhere.